src/version/version.rest.module.ts

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Module } from '@nestjs/common';
+import { PrismaModule } from '../prisma/prisma.module';
+import { VersionController } from './version.controller';
+import { VersionService } from './version.service';
+
+@Module({
+  controllers: [VersionController],
+  exports: [VersionService],
+  imports: [PrismaModule],
+  providers: [VersionService],
```

Comment:
For consistency, what do you think about moving the Prisma related services to be within their own module (ex. `versions.module.ts`) and then leaving the controller here?